tests/runtime/basic

```diff
@@ -285,11 +285,13 @@ NAME variable strings are memset
 PROG BEGIN { $x = "xxxxx"; $x = "a"; @[$x] = 1; $y = "yyyyy"; $y = "a"; @[$y] = 1; printf("len: %d\n", len(@)); exit(); }
 EXPECT len: 1
 TIMEOUT 1
+SKIP_VERIFY_LLVM_IR 14,15
```

Comment:
Nit: maybe it would be good to add a comment here with either an issue or some details as to why this is needed (re: the segfault and dbg info)